<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHomepage</class>
 <widget class="QMainWindow" name="UserHomepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>788</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1189</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>751</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/Background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1191</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 192);
 
    padding: 10px;
 border: 1px solid #faf4ec; 
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelProfile">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/ic_profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogout">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/ic_logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdmin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 160, 175);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1191</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Tab đang được chọn (active) */
QTabBar::tab:selected {
	min-width: 300px;     /* Độ rộng tối thiểu của mỗi tab */
    min-height: 35px;
    background-color: #b96e7f;     /* màu hồng đậm khi chọn */
    color:#e8ddde ;                 /* chữ trắng khi chọn */
    font-weight: bold;
	border-radius: 10px;
	font: 16pt &quot;MS Shell Dlg 2&quot;;
   font-weight: bold;
}

/* Tab chưa được chọn */
QTabBar::tab:!selected {
	min-width: 300px;     /* Độ rộng tối thiểu của mỗi tab */
    min-height: 35px;
    background-color: #e8ddde;    /* hồng nhạt khi chưa chọn */
    color: #b96e7f;               /* chữ nâu đỏ */
    font-weight: normal;
	border-radius: 10px;
	font: 16pt &quot;MS Shell Dlg 2&quot;;
   font-weight: bold;
}

/* Khi rê chuột vào tab (hover) */
QTabBar::tab:hover {
    background-color: #dda0af;    /* hồng sáng hơn khi hover */
}

/* Viền xung quanh vùng nội dung tab */
QTabWidget::pane {
    border: 2px solid #dda0af;
    border-radius: 10px;
}

</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="tabHomepage">
     <attribute name="icon">
      <iconset>
       <normaloff>../images/ic_home.png</normaloff>../images/ic_home.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Homepage</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1171</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8ddde;   
color: rgb(104, 83, 92);        
font: 13pt &quot;MS Shell Dlg 2&quot;;

</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1169</width>
         <height>579</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRecomendations">
     <attribute name="icon">
      <iconset>
       <normaloff>../images/ic_recomendation.png</normaloff>../images/ic_recomendation.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Recommendations</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(104, 83, 92);        
font: 16pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/ic_date.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(104, 83, 92);        
font: 16pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Store</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/ic_store.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditDate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>611</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 190, 193);
color: rgb(104, 83, 92);        
font: 14pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
border-radius: 15px;</string>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2013</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="currentSectionIndex">
       <number>2</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxStore">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>611</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 190, 193);
color: rgb(104, 83, 92);          
font: 14pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
border-radius: 15px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPredictRecomendation">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>40</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #b96e7f;   
    color:#e8ddde ;                 
	border-radius: 10px;
font: 15pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Recommendation</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../images/ic_recomend.png</normaloff>../images/ic_recomend.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>811</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(185, 110, 127);
background-color: rgb(250, 244, 236);
border: 1px solid #faf4ec; 
font: 75 18pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>811</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 160, 175);
 color: rgb(104, 83, 92);        
font: 18pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Top 5 Best-Selling Products</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelHello">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(104, 83, 92); 
font: 12pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHello_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: rgb(104, 83, 92);        
font: 18pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>labelAdmin</zorder>
   <zorder>labelProfile</zorder>
   <zorder>labelLogout</zorder>
   <zorder>label_5</zorder>
   <zorder>tabWidget</zorder>
   <zorder>labelHello</zorder>
   <zorder>labelHello_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
